spec/system/admin/enterprises_spec.rb

```diff
@@ -774,6 +774,13 @@
               expect(page).to have_content("Custom tab content")
             end
 
+            it "enable the update button on custom tab content change" do
+              fill_in_trix_editor "custom_tab_content", with: "Custom tab content changed"
+              within "save-bar" do
+                expect(page).to have_button("Update", disabled: false)
+              end
```

Comment:
Can we go further and test that the content can be saved to the db?

Reply:
Yes! Done ;)

Reply:
👏